README.md

```diff
@@ -9,6 +9,11 @@
 - JAVASCRIPT
 - LINTERS
 
+## Live Demo
+[coming soon](#)
+
+![Screenshot](src/assets/SCREENSHOT1.PNG)
+
```

Comment:
- [ ] Kİndly fix the broken link in the license section. Also, fill the acknowledgment section proper way. Texts are from the template.
![image](https://user-images.githubusercontent.com/38635158/176980266-cfdde9e8-a285-452d-ad27-5801ed95a1c3.png)